package.json

```diff
@@ -14,8 +14,8 @@
     "format": "prettier --write 'src/**/*.{js,json,css,scss,md,mdx,yaml}'"
   },
   "dependencies": {
-    "@carbon/charts": "^0.16.15",
-    "@carbon/charts-react": "^0.16.15",
+    "@carbon/charts": "^0.16.22",
+    "@carbon/charts-react": "^0.16.22",
```

Comment:
it looks like there have been a few releases since `0.16.22`, would we want to bring in the latest release?